sqlserver/tests/common.py

```diff
@@ -32,11 +34,14 @@ def get_local_driver():
 CUSTOM_METRICS = ['sqlserver.clr.execution', 'sqlserver.db.commit_table_entries', 'sqlserver.exec.in_progress']
 EXPECTED_METRICS = [
     m[0]
-    for m in SQLServer.INSTANCE_METRICS
-    + SQLServer.TASK_SCHEDULER_METRICS
-    + SQLServer.DATABASE_METRICS
-    + SQLServer.DATABASE_FRAGMENTATION_METRICS
-    + SQLServer.FCI_METRICS
+    for m in chain(
```

Comment:
What is the difference between using `+` for concatenation and `chain()`?

Reply:
The `+` operator will incrementally create new strings with each addition, taking additional memory.  `chain` will just sequentially iterate through each iterator without creating new objects.

For test code, it's not a big deal, but this section was getting a little long.